rtb/piano_di_progetto_ver0.2.0.typ

```diff
@@ -220,12 +227,88 @@ Le attività che il gruppo si è preoccupato di concludere durante queste due se
+Gli analisti evidenziano le difficoltà e i dubbi relativi alla comprensione dei casi d'uso del prodotto richiesto dal proponente, che hanno però trovato risposta dalla riunione di fine primo _sprint_ da parte del proponente e che verranno approfonditi in quello successivo. Incentivano inoltre a lavorare in maniera più collaborativa alla fase di analisi e non limitarsi semplicemente all'esecuzione delle attività di analisi in maniera parallela.
 
 I verificatori suggeriscono infine di incaricare più di una persona alla verifica dei documenti ad alto contenuto, quali possono essere le Norme di Progetto e il Piano di Progetto, al fine di garantire una verifica ottimale.
 
 
+#pagebreak()
+
+=== Sprint 2
+- *Inizio*: 28/11/2024
+- *Fine*: 13/12/2024
+- *Rischi attesi*: #link(<RO-2>)[RO-2], #link(<RO-3>)[RO-3], #link(<RT-1>)[RT-1]
+
+==== Attività dello sprint
+//Quello che è stato fatto o quello che è da fare ?
+Le attività che il _team_ si è prefissato di portare a termine durante il secondo _sprint_ sono le seguenti:
+- Redazione verbale esterno ed interno del 27/11/2024.
+- Creare una prima versione del simulatore dati.
+  - È stato deciso di dedicare la prima settimana dello _sprint_ allo studio e approfondimento delle tecnologie: #rifGlossario("Docker"), #rifGlossario("Apache Kafka"), #rifGlossario("Apache Nifi") e #rifGlossario("PostgresSQL").
+  - La seconda settimana sarà invece riservata all'implementazione concreta del simulatore e del _database_ dove verranno salvati i dati generati, grazie all'utilizzo delle tecnologie studiate.
+- Avanzamento nel documento Analisi dei Requisiti, in particolare: 
+  - Migliorando i vecchi casi d'uso dopo aver risolto alcuni dubbi in seguito al confronto con l'azienda.
+  - Scrivendo nuovi casi d'uso.
+- Continuazione stesura Piano di Progetto, in particolare:
+  - Scrittura retrospettiva primo _sprint_.
+  - Scrittura pianificazione secondo _sprint_.
+- Inizio stesura del Piano di Qualifica.
+- Modifica alle Norme di Progetto, per via del nuovo aggiornamento riguardo al sistema di _versioning_ (spiegazione completa nel VI_2024-11-27).
+- Modifica delle versioni dei seguenti documenti, per coerenza con il nuovo sistema di versionamento:
+  - Glossario
+  - Analisi dei Requisiti
+  - Piano di Progetto
+  - Norme di Progetto
+- Modifica al _template_ ed al _test_ del Glossario.
+  - La modfica al _template_ vedrà l'implementazione di:
+    - Una funzione per automatizzare la generazione dei grafici relativi al preventivo ed al consuntivo orario richiesti per ogni sprint, nel Piano di Progetto.
+    - Aggiunta dell'indice delle immagini e delle tabelle, generato automaticamente, su tutti i file.
+  - La modifica al _test_ del glossario è necessaria in quanto, attualmente, segnala dei falsi positivi. Per esempio quando abbiamo parole appartenenti al glossario contenute in altre parole, oppure la segnalazione di termini del Glossario che in realtà sono comandi #rifGlossario("Typst").
```

Comment:
- la prima occorrenza di "Glossario" la metterei in maiuscolo, la seconda non saprei, chiedo anche a @MilanRiccardo03

Reply:
Effettivamente in 3.1.8 delle NdP abbiamo:
> Maiuscole:
• Nomi dei documenti (ad esempio «Norme di Progetto»).

Quindi immagino Glossario vada in maiuscolo tutte e due le volte, my bad.

Reply:
se ci si riferisce al documento sì, ci sarebbe la sottigliezza di distinguere glossario in termini generali da Glossario documento però nel dubbio lascerei in maiuscolo